streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -199,6 +219,8 @@
 
     /**
      * Convert this stream to a new instance of {@link KStream}.
```

Comment:
This is not introduced by this PR, but I think it is more natural as "a new {@link KStream} instance."

Also how about "that contains" (or containing, again either is fine and we just pick one throughout the class files) "the same records as this {@link KTable}. The records are no longer treated as updates on the primary-keyed table anymore, but rather as normal key-value pairs in a record stream." Ditto below.